reference/appearance.md

```diff
@@ -23,10 +24,14 @@ lights are ignored during rendering of the object that references this
 [Appearance](#appearance)) and the unlit object color is (1,1,1).
 
 - The `texture` field, if specified, shall contain an
-[ImageTexture](imagetexture.md) node. If the `texture` node is NULL or the
+[ImageTexture](imagetexture.md) node or a [MultiTexture](multitexture.md) node.
+If the `texture` node is NULL or the
 `texture` field is unspecified, the object that references this
```

Comment:
I removed the `unspecified` keyword from all the field documentation of the Appearance nodes.